kiosk/tests/rules/royalty_rule_tests.move

```diff
@@ -20,15 +20,12 @@ module kiosk::royalty_rule_tests {
         royalty_rule::add(&mut policy, &cap, 0, 0);
 
         let request = policy::new_request(test::fresh_id(ctx), 100_000, test::fresh_id(ctx));
-        let payment = coin::mint_for_testing<SUI>(2000, ctx);
+        let payment = coin::mint_for_testing<SUI>(0, ctx);
```

Comment:
Can we do a nonzero payment here for realism? I would expect us to abort on zero-valued payments.